tests/e2e/integration/networks.js

```diff
@@ -44,71 +44,68 @@ describe('Test cases for networks page', () => {
       .should('not.exist');
   });
 
-  // it('Can add, select, edit, delete new network, can not add network with the same name', () => {
-  //   cy.addNetwork(
-  //     'Newnetwork',
-  //     defaultNetwork.url,
-  //     defaultNetwork.middlewareUrl,
-  //     defaultNetwork.compilerUrl,
-  //   )
-  //     .get('[data-cy=close]')
-  //     .click()
-  //     .openNetworks()
-  //     .get('[data-cy=more]')
-  //     .click()
-  //     .get('[data-cy=edit]')
-  //     .should('be.visible')
-  //     .click()
-  //     .get('[data-cy=network] input')
-  //     .should('have.value', 'Newnetwork')
-  //     .get('[data-cy=url] input')
-  //     .should('have.value', defaultNetwork.url)
-  //     .clear()
-  //     .type(defaultNetwork.url)
-  //     .get('[data-cy=middleware] input ')
-  //     .should('have.value', defaultNetwork.middlewareUrl)
-  //     .clear()
-  //     .type(defaultNetwork.middlewareUrl)
-  //     .get('[data-cy=connect]')
-  //     .click()
-  //     .get('[data-cy=network-url]')
-  //     .eq(-1)
-  //     .should('contain', defaultNetwork.url)
-  //     .get('[data-cy=network-middleware]')
-  //     .eq(-1)
-  //     .should('contain', defaultNetwork.middlewareUrl)
-  //     .goBack()
-  //     .goBack()
-  //     .goBack()
-  //     // TODO should be aligned with the new home page changes
-  //     // .get('.transaction-list', { timeout: 8000 })
-  //     // .should('exist')
-  //
-  //     .openNetworks()
-  //     .get('[data-cy=to-add]')
-  //     .click()
-  //     .enterNetworkDetails(
-  //       'Newnetwork',
-  //       defaultNetwork.url,
-  //       defaultNetwork.middlewareUrl,
-  //       defaultNetwork.compilerUrl,
-  //     )
-  //     .get('[data-cy=connect]')
-  //     .click()
-  //     .buttonShouldBeDisabled('[data-cy=connect]')
-  //     .get('[data-cy=cancel]')
-  //     .click()
-  //
-  //     .get('[data-cy=more]')
-  //     .click()
-  //     .get('[data-cy=delete]')
-  //     .should('be.visible')
-  //     .click()
-  //     .get('[data-cy=networks] .network-row')
-  //     .should('have.length', 2)
-  //     .get('[data-cy=networks] .network-row')
-  //     .eq(1)
-  //     .find('.checkmark')
-  //     .should('have.class', 'checked');
-  // });
+  it('Can add, select, edit, delete new network, can not add network with the same name', () => {
+    cy.addNetwork(
+      'Newnetwork',
```

Comment:
I know I'm picking on, but maybe you could do name this `NewNetwork`? :)
I have spellcheck plugin installed to my IDE and it's complaining about not existing words.